app/components/Comments/Comments.js

```diff
@@ -17,8 +17,11 @@ export default class Comments extends Component {
   constructor(props) {
     super(props);
     const ds = new ListView.DataSource({ rowHasChanged: (r1, r2) => r1 !== r2 });
-    this.state = { dataSource: ds.cloneWithRows(this.props.comments) };
-    this.props.actions.fetchComments();
+    this.state = { dataSource: ds.cloneWithRows(props.comments) };
+  }
+
```

Comment:
this.state = { dataSource: new ListView.DataSource({ rowHasChanged: (r1, r2) => r1 !== r2 });